services/harmony/env-defaults

```diff
@@ -135,6 +135,18 @@ MAX_DATA_OPERATION_CACHE_SIZE=512000000
 # Token Cache TTL in milliseconds
 TOKEN_CACHE_TTL=300000
 
+# Maximum size (in bytes) of the cache for CMR searches
+EDL_CACHE_SIZE=256000000
+
+# EDL Cache TTL in milliseconds
+EDL_CACHE_TTL=600000
+
+# Maximum size (in bytes) of the cache for CMR searches
+CMR_CACHE_SIZE=1024000000
```

Comment:
I think 1GB should be ok but probably want to sanity check we have that much free memory - I think we explicitly set a heap size to ~3GB.